### DISCUSSION TITLE
👋  Who else is using React Flow?

### DISCUSSION THREAD

**Discussion by cliffordfajardo** [xyflow/xyflow#3588] [Show and tell]
It would be nice to include a list of organisations that use `react-flow` in the docs.

**Please include:**
1. Organisation name
2. Link to organization site
3. If possible, you can attach a codesandbox,  public product screenshots or demo URL page.

CC @moklick @chrtze  - please take a look at details dropdown below 🙏 

<details>
  <summary>Details (this is inspired by what I saw in this repo ↓</summary>
  
Graphin (another data viz library) has an issue pinned on their issues page allowing users to post how they're using the library. - https://github.com/antvis/Graphin/issues

![image](https://user-images.githubusercontent.com/6743796/155409436-cdc266aa-695b-4e90-8f3c-600e1a7a8ea7.png)

🤔  Unfortunately, I don't think Github Disussions allows pinning of content 
</details>

**Comment by cliffordfajardo** [comment]
Organisation name: 
- LinkedIn
- Website: linkedin.com

**Background Context**
Use Case: I'm using it for an internal tools, to help us visualize our data center infrastructure (ex: devices on a network & what its connected to)
Why I decided to use `react-flow-renderer`:
   - for my use cases I wanted custom DOM (React)
   - I was only every going to render at most 100 items in a graph/view, so Canvas was overkill & limiting here (no custom DOM). I have not tried rendering 100+ items yet, but in the docs it 50-low hundreds looks feasible ([stress test example](https://reactflow.dev/examples/stress/))
   - the API of this library is nice, typescript based & docs are nice

<img width="1790" alt="CleanShot 2022-02-23 at 13 06 35@2x" src="https://user-images.githubusercontent.com/6743796/155408658-49aa4d54-8dc9-47ac-86db-e5ca3814fbeb.png">

https://user-images.githubusercontent.com/6743796/155408742-b3fe8134-c625-4754-ad1b-afb8a603fcfc.mp4


ℹ️ In case anyone is new to their data visualization journey and wondering when it might be best to use SVG/DOM vs Canvas/WebGL etc, this video really helped me identify my use case & arrive at the conclusion that `react-flow-renderer` was a good fit for my use cases.
- YouTube Video: [SVG, Canvas, and WebGL: The Power, the Beauty, and the Cost](https://youtu.be/wjtfQ8NI2T8?t=266)

**Comment by moklick** [comment]
Hey @cliffordfajardo 

this is a very good idea! Thanks, we will pin it :) I think you meant ".. that use React Flow ..." instead of "..that use graphin ...".

**Comment by joshuataylor** [comment]
I found this project through https://github.com/re-data/re-data

They use it for their lineage - https://docs.getre.io/ui-latest/#/graph

**Comment by jdsamar** [comment]
does anyone know how to change properties of drag and drop components in react-flow-renderer

**Comment by cliffordfajardo** [comment]
Hi @jdsamar 
The community would be happy to help;  Would you be able to ask this question in the new Q&A section of this github repository located here?
- https://github.com/wbkd/react-flow/discussions/new?category=q-a

Asking in the [Q&A section ](https://github.com/wbkd/react-flow/discussions/new?category=q-a)of the github repository will help others in the future as well 🙏 

This also might not be the most appropriate github thread for this type of question 😉

**Comment by Ovyerus** [comment]
Not an org, but I made a database schema visualiser using React Flow!

Repo: https://github.com/Ovyerus/prismaliser
Site: https://prismaliser.app/

**Comment by cliffordfajardo** [comment]
Some of these open source project were shared in the discord

## chaiNNer ([Github](https://github.com/joeyballentine/chaiNNer))
> A flowchart/node-based image processing GUI aimed at making chaining image processing tasks (especially those done by neural networks) easy, intuitive, and customizable.
> No existing GUI gives you the level of customization of your image processing workflow that chaiNNer does. Not only do you have full control over your processing pipeline, you can do incredibly complex tasks just by connecting a few nodes together.

![image](https://user-images.githubusercontent.com/6743796/159724605-04caa2ad-9ccf-4832-9afc-49ad8f67ace8.png)


## bitFlow ([Github](https://github.com/openpatch/bitflow))
> Bitflow a library for creating and conducting dynamic flow-based assessments.

![CleanShot 2022-03-23 at 07 43 20](https://user-images.githubusercontent.com/6743796/159725874-aeb5dc73-be9e-46f5-b310-cd781001d4cf.png)


## Audio tool ([Github](https://github.com/Nearoo/audio-tool))
> A node-based DAW built on react

![image](https://user-images.githubusercontent.com/6743796/159726098-b2adf59b-b74e-4c3a-8d62-aa6bc46c5400.png)


## prereq-flow  ([Github](https://github.com/awu43/prereq-flow))
> Prereq Flow is an unofficial course planning aid for University of Washington students that visualizes courses and prerequisites in undergraduate degrees.

https://user-images.githubusercontent.com/6743796/159726863-bdf66ad9-b0b6-48a2-851b-06645056f686.mp4


## Concepts from people in community (no source code at this time)

![image](https://user-images.githubusercontent.com/6743796/159727548-49e5cfff-2ffb-4a4e-9849-1a921d3c9b57.png)


## doubleloop.app ([Website](https://doubleloop.app/))
> DoubleLoop helps product teams visualize their strategy, track the performance of their initiatives, and get better outcomes.

![CleanShot 2022-03-23 at 07 56 46](https://user-images.githubusercontent.com/6743796/159728738-3164798f-7de7-4d06-887f-eb88079ca689.png)

**Comment by dzikoysk** [comment]
I see there are some links to open-source projects, so I can also mention one more - I'm using React Flow in a read-only component on the Reposilite's landing page:
* Website: https://reposilite.com
* Sources: https://github.com/dzikoysk/reposilite/tree/main/reposilite-site/components/landing/scenarios

**Comment by longnc100500** [comment]
![image](https://user-images.githubusercontent.com/48049372/170863728-d1d3432b-6b41-4e3b-b2ad-16d3bbaaa8a0.png)
<img width="1255" alt="image" src="https://user-images.githubusercontent.com/48049372/170863792-3095ecc8-8fa4-4e0a-b159-c27e7c7f7491.png">

this is my mindmap project base on reactflow

**Comment by doflo-dfa** [comment]
still a work in progress but should be soft launching in the next month or two
<img width="2967" alt="image" src="https://user-images.githubusercontent.com/77824565/185336664-31ef775d-2c33-45b0-945c-3ed1f98db109.png">

**Comment by spaceymonk** [comment]
Here is my open-source project, FlowerApp. It enables you to create, run, analyze and export (as PNG and pseudo-code) flowcharts.

![ss](https://user-images.githubusercontent.com/44543385/187373585-918ab865-eea2-4784-afc8-9d1d174e07f2.jpg)

Repo: https://github.com/spaceymonk/flower-app
Web site: https://web.itu.edu.tr/~ozkanbe19/flower-app/

**Comment by ozovalihasan** [comment]
Hi everyone :wave:,

Thanks to all contributors for this amazing library :rocket::rocket::rocket:

I developed a tool([EAD, Entity Association Diagram)](https://github.com/ozovalihasan/ead) to make the prototyping process of a Ruby on Rails project faster and easier. It is an open-source project.

![example-ead](https://user-images.githubusercontent.com/44648011/193152896-82b682a4-24e3-4d12-be14-edeb9c9003ef.png)

**Comment by dsilva01** [comment]
My project is a flowchart convert, you can create, run, save, and export (as java code and PNG)

 
![Screenshot from 2022-10-10 00-53-30](https://user-images.githubusercontent.com/65828956/194785246-c7127df8-35a3-49e0-9139-9eb2190aa40b.png)

 * Sources: https://github.com/dsilva01/convert-flowchart
 * Website: https://flowchart.desideriosilva.com/

**Comment by dsilva01** [comment]
@spaceymonk, my project is similar to yours

**Comment by Sarfraz-droid** [comment]
Working on a project which allows users to create code-logic using a visual editor


https://user-images.githubusercontent.com/73013838/209326134-15b4fae6-e944-4bc0-be6a-0eb0d7322295.mp4



- Repo : https://github.com/Sarfraz-droid/JS-Blueprints
- Website : https://jsblueprints.sarfrazalam.in/

**Comment by makaroni4** [comment]
Thank you SO MUCH @moklick and @chrtze 🙌 I've tried all graph/flow JS libraries and React Flow is light years ahead of everyone. I really appreciate that the community version is open-sourced and MIT ❤️ 

I run a Data Analysis course called [SQL Habit](https://www.sqlhabit.com/) and my students often ask for schema visualization. Here it is:

![Screenshot 2023-01-19 at 21 49 40](https://user-images.githubusercontent.com/768070/213556695-168e92c3-5d7a-444a-9444-a1c7c402fecd.png)

The visualizer is also open-sourced, anyone can clone the repo and create their own schema visualization. 🍻 

🔗 Website: https://sqlhabit.github.io/sql_schema_visualizer/
🔗 Github repo: https://github.com/sqlhabit/sql_schema_visualizer

**Comment by kanishmalviya** [comment]
Organization name: Futwork
Website: www.futwork.com
Usecase: Internal tool to create conversation flow and nodes are render in APP as screens.
Added screenshot which is 1/4 part of entire canvas.

![Futwork Reat Flow](https://user-images.githubusercontent.com/29313203/225109803-11970bce-d09b-4c46-a281-0f73bf9f391b.png)

**Comment by aleksbobic** [comment]
Thanks for making this amazing library! It saved me a lot of time and enabled me to create some features and functionalities that would otherwise be very hard to implement! 😄 

I'm using it in my PhD project called Collaboration Spotting X, which is a network-based visual analytics and information retrieval tool for analyzing tabular data using networks. I use react flow, particularly for the history component, interactive graph schema as well as an advanced search component.

![Collaboration Spotting X - Preview](https://github.com/aleksbobic/csx/blob/master/cover.gif?raw=true)

Website: [csxp.me](https://www.csxp.me](https://www.csxp.me)
GitHub: [https://github.com/aleksbobic/csx](https://github.com/aleksbobic/csx)

**Comment by RichSchulz** [comment]
1. Taktile
2. https://www.taktile.com
<img width="1507" alt="Screenshot 2023-05-11 at 18 35 54" src="https://github.com/wbkd/react-flow/assets/49609334/62943669-b664-4025-9ede-b698317438d0">

**Comment by nixxholas** [comment]
SolanaFM
https://solana.fm
